Resources/Prototypes/Entities/Items/Consumables/drinks.yml

```diff
@@ -635,7 +616,7 @@
     sprite: Objects/Drinks/ginvodkaglass.rsi
 
 - type: entity
-  parent: DrinkBase
+ parent: DrinkBase
```

Comment:
?

Reply:
holy shit that would've been bad good catch